### DISCUSSION TITLE
Netdata Disable Postgres Module Configured Alarm

### DISCUSSION THREAD

**Discussion by alex-iocurrents** [netdata/netdata#18819] [Q&A]
I have a netdata alarm [that I would like to disable](https://github.com/netdata/netdata/blob/master/src/health/guides/postgres/postgres_db_cache_io_ratio.md). I specifically think I have an edge case that makes this alarm trigger even when it's not useful in our case. More importantly, the alarm flaps from warning to ok several times an hour.

Unfortunately, looking at the configuration, the PostgreSQL module seems to be [all or nothing](https://github.com/netdata/netdata/blob/5813cf2495a4165bf203135599b67fb3f107ef6e/src/go/plugin/go.d/modules/postgres/config_schema.json) when it comes to alarms. I'm wondering if there's something I'm missing here are is there somewhere else in the product that would let me disable or configure this particular alarm in some way?

**Comment by ilyam8** [comment]
Hi, @alex-iocurrents. You can disable any alarm by 

- adding the `host labels: _hostname=!*` line to it (and restart Netdata).
- using dyncfg.

<details>
<summary>Disable via UI</summary>

https://github.com/user-attachments/assets/eae3fefe-7319-4dbd-93b4-e1c5f2c1d7e4

</details>

**Reply by alex-iocurrents** [reply]
Does the `host labels: _hostname=!*` solution mean something like

```
host labels: _hostname=!*
```

translate to a `go.d/postgres.conf` file that looks like

```
jobs:
  - name: local
    dsn: 'host=/var/run/postgresql dbname={{ seasensor_database_name }}  user=netdata'
    timeout: 4
host:
  labels: _hostname=!*
```
and if so how would this match specifically to `postgres_db_cache_io_ratio` without disabling other alarms in the module?

**Reply by ilyam8** [reply]
> Does the host labels: _hostname=!* solution mean something like

Yes. But you need to add this line to the [alert](https://github.com/netdata/netdata/tree/master/src/health/health.d) you want to disable.

Or

1. Open `netdata.conf`.
2. Go to `[health]`.
3. Uncomment and update `enabled alarms = !postgres_db_cache_io_ratio *`.
4. Restart netdata service.
